httpbp/errors.go

```diff
@@ -65,7 +65,7 @@ const (
 	DefaultErrorTemplateName = "httpbp/error"
 )
 
-// ClientMiddleware errors are returned if a known error is occuring during the middleware processing.
+// ClientMiddleware errors returned if a known error is occuring during the middleware processing.
```

Comment:
Hmm, this symbol name doesn't match the var?

Do you mean

```
// Well-known errors for middleware layers.
var (
  // ErrConcurrencyLimit is returned by Foo if there are too many requests in flight
  ErrConcurrencyLimt = ...
)
```

Reply:
Something I borrowed from the way GoDoc is provided for errors in `thriftbp` but worth adding a separate comment: https://github.com/reddit/baseplate.go/blob/master/thriftbp/errors.go#L47-L52